src/library/Hooks/usePayouts/fetchDataForEra.ts

```diff
@@ -0,0 +1,98 @@
+import { ApiPromise } from '@polkadot/api';
+import BN from 'bn.js';
+import { DECIMALS } from './consts';
+import { EraData } from './types';
+
+/**
+ * Warning: Lots of "ts-ignore"s due to incorrect api library's types.
```

Comment:
This is a bummer since with the amount of data to be fetched, it's easy to overlook a possibility for a runtime error. I wrapped the payouts graph component with an error boundary in the `Overview` page so that if it happens, it doesn't blow up the entire page.